var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -30,7 +30,89 @@ class Tasmanian(CMakePackage):
     ApproximatioN is a robust library for high dimensional integration and
     interpolation as well as parameter calibration."""
 
-    homepage = "http://tasmanian.ornl.gov"
-    url      = "http://tasmanian.ornl.gov/documents/Tasmanian_v5.0.zip"
+    homepage = 'http://tasmanian.ornl.gov'
+    url      = 'https://github.com/ORNL/TASMANIAN/archive/v5.1.tar.gz'
 
-    version('5.0', '4bf131841d786033863d271739be0f7a')
+    version('develop', branch='master',
+            git='https://github.com/ORNL/TASMANIAN.git')
+
+    version('5.1', '5d904029a24470a6acf4a87d3339846e',
+            url='https://github.com/ORNL/TASMANIAN/archive/v5.1.tar.gz')
```

Comment:
This URL shouldn't be necessary, if you remove it, Spack will use the default package-level URL. You can use `spack info` or `spack fetch` to confirm.

Reply:
Hi, I am not sure I follow this one. If I do not specify the `url` for version 5.1, then spack searches the `homepage` as opposed to the package level `url`, e.g., 
```
http://tasmanian.ornl.gov/documents/Tasmanian_v5.1.zip
```
The homepage is not the same as the `git` repo.